tf/chunkparser.py

```diff
@@ -28,10 +69,14 @@
 import gzip
 from select import select
 
+V6_VERSION = struct.pack('i', 6)
 V5_VERSION = struct.pack('i', 5)
 CLASSICAL_INPUT = struct.pack('i', 1)
 V4_VERSION = struct.pack('i', 4)
 V3_VERSION = struct.pack('i', 3)
+# jhorthos query - not sure i understand Struct format strings fully B? b?
+# followed by 15 floats, 1 uint_32, 2 uint_16, and 64 (4x16) reserved
+V6_STRUCT_STRING = '4si7432s832sBBBBBBBBfffffffffffffffIHH4H'
```

Comment:
While Borg changed v5 result from signed to unsigned dummy in v6, I'm not sure we should respect that here - since it would allow for simplification of convert_v6_to_tuple. (And dummy is unused so interpreting it as per the v6  spec has no actual effect)